packages/components/src/styles/container/container.css

```diff
@@ -20,4 +20,91 @@
   &--padding-sm {
     @apply px-6 py-4;
   }
+
+  --triangle-background: white;
+
+  &--triangle-top::before,
+  &--triangle-right::before,
+  &--triangle-bottom::before,
+  &--triangle-left::before {
+    @apply absolute block content-[''] border-solid border-transparent border-[14px];
+  }
+
+  &--triangle-top::before {
+    @apply left-[calc(50%-14px)] top-0 border-t-[var(--triangle-background)];
+  }
+  &--triangle-right::before {
+    @apply top-[calc(50%-14px)] right-0 border-r-[var(--triangle-background)];
+  }
+  &--triangle-bottom::before {
+    @apply left-[calc(50%-14px)] bottom-0 border-b-[var(--triangle-background)];
+  }
+  &--triangle-left::before {
+    @apply top-[calc(50%-14px)] left-0 border-l-[var(--triangle-background)];
+  }
+
+  &--triangle-top-border,
+  &--triangle-right-border,
+  &--triangle-bottom-border,
+  &--triangle-left-border {
+    &::after,
+    &::before {
+      @apply absolute block content-[''] border-solid border-transparent;
+    }
+    &::after {
+      @apply border-[14px];
+    }
+    &::before {
+      @apply border-[15px];
+    }
```

Comment:
I don't feel super great with the number of arbitrary values used within Tailwind. 
We have updated guidelines to use plain css for custom values inside css literals (this is to avoid bloating the Tailwind.css bundle size). I am not sure if it applies here since it was referring to the css we use inside components and not styles, but I imagine the same concept applies here.

Here's the section I'm referring to --> https://github.com/solid-design-system/solid/blob/main/CONTRIBUTING.md#we-only-use-tailwindcss-or-solid-styles.  

What do you think @karlbaumhauer @abudd1094 ?

Reply:
Oh, wow just saw this was merged haha. 
I'd say we can follow-up on this on another ticket if it makes sense to do so.